pkg/storage/tsdb/config.go

```diff
@@ -172,6 +172,11 @@ type TSDBConfig struct {
 
 	// How often to check for idle TSDBs for closing. DefaultCloseIdleTSDBInterval is not suitable for testing, so tests can override.
 	CloseIdleTSDBInterval time.Duration `yaml:"-"`
+
+	// Highly experimental out of order metrics support
+	OOOAllowance time.Duration `yaml:"tsdb_ooo_allowance" category:"experimental"`
+	OOOCapMin    int           `yaml:"tsdb_ooo_cap_min" category:"experimental"`
+	OOOCapMax    int           `yaml:"tsdb_ooo_cap_max" category:"experimental"`
```

Comment:
These are already nested within `tsdb`
```suggestion
	OOOAllowance time.Duration `yaml:"ooo_allowance" category:"experimental"`
	OOOCapMin    int           `yaml:"ooo_cap_min" category:"experimental"`
	OOOCapMax    int           `yaml:"ooo_cap_max" category:"experimental"`
```

Reply:
Addressed in https://github.com/grafana/mimir/pull/1757/commits/7ba81c38f452e07b85d222169a489a2f56bddc99